jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -104,7 +106,7 @@ public void initColumns(final ListTable<ProcessInstanceSummary> extendedPagedTab
         extendedPagedTable.addSelectionIgnoreColumn(actionsColumnMeta.getColumn());
         extendedPagedTable.addSelectionIgnoreColumn(errorCountColumn);
 
-        final List<ColumnMeta<ProcessInstanceSummary>> columnMetas = new ArrayList<ColumnMeta<ProcessInstanceSummary>>();
+        final ArrayList<ColumnMeta<ProcessInstanceSummary>> columnMetas = new ArrayList<ColumnMeta<ProcessInstanceSummary>>();
```

Comment:
no need to change type in here

Reply:
if do not use it, columnMetas.add will throw exception: means AbstractList do not have add .

Reply:
you still using _ArrayList_ as implementation, just casting to the interface, it doesn't make much sense to me.